pkgs/by-name/ru/rutabaga_gfx/package.nix

```
@@ -40,8 +40,22 @@ stdenv.mkDerivation (finalAttrs: {
     })
     # Install the dylib on Darwin.
     ./darwin-install.patch
+    # Patch for libc++, drop in next update
+    # https://issuetracker.google.com/issues/392934337
```

Would make more sense to link to the commit the patch is adapting, rather than indirectly through the closed issue.

---

Maybe, though it looks like the commit link is a part of the patch before the query?

---

The query part of the URL. Before the `?`.

---

Right, but that URL is currently not included here now that the fetchpatch is gone?  I think we should do

```suggestion
    # https://chromium.googlesource.com/crosvm/crosvm/+/8ae3c23b2e3899de33b973fc636909f1eb3dc98c
```

---

I forgot about that lol.